.travis.yml

```
@@ -1,11 +1,9 @@
 language: ruby
 
 rvm:
-  - 2.4.5
-  - 2.5.0
-  - 2.5.1
-  - 2.5.2
-  - 2.5.3
+  - "2.5"
+  - "2.6"
+  - "2.7"  
```

```suggestion
  - 2.5
  - 2.6
  - 2.7
```
Looking to travis documentation I like more this way to write it.
https://docs.travis-ci.com/user/build-matrix/#matrix-expansion

What do you think?

---

I'll make the change; this detail is mostly "avoid Float" (for non-Travis YAML tools to be able to consume the file).

---

non-Travis? but this is settings file for travis, right?

---

Change applied! ✨